src/components/BookInput.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { ACTIONS } from '../redux/books/books';
+
+function BookInput() {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const [type, setType] = useState('');
+
+  // console.log(books);
```

Comment:
* [OPTIONAL] kindly remove all `commented codes` instances from your codebase as it makes your code base bulky and unprofessional